cli/install/install.go

```diff
@@ -304,10 +304,10 @@ func programDependenciesInstalled(program string) bool {
 				" are installed.\n"+
 				"Proceed without checking?")
 			if !answer || err != nil {
-				return false
+				return err
```

Comment:
Suppose this will return nil and we will continue installation. Which is not what a user expects.

Reply:
Fixed it. Returning `false, error` allows to handle it using bool type and return error if it was occured.